### DISCUSSION TITLE
Where was normal-foreground defined before 2053818?

### DISCUSSION THREAD

**Discussion by Aster89** [davatorium/rofi#1677] [Q&A] [answered]
Out of curiosity, before https://github.com/davatorium/rofi/commit/205381846637426dfe9a30056bac9df47503532a, `sidebar.rasi` was making use of `normal-foreground` which was not defined in the same file. Where was it?

The reason I'm asking is that my `~/.config/rofi/sidebar.rasi` has got old, but only today I've got it broken, telling me
> Validating the theme failed: the variable 'normal-foreground' in `textbox-prompt-colon { text-color: var(normal-foreground);}` failed to resolve.

**Comment by DaveDavenport** [comment, marked as answer]
it wasn't.    in the past rofi 'ignored' (and resulted in broken themes)  undefined variables. 
In 1.7.4 rofi tells you about it being broken.

**Reply by DaveDavenport** [reply]
see release notes. https://github.com/davatorium/rofi/releases/tag/1.7.4
